geniza/corpus/admin.py

```diff
@@ -371,15 +385,19 @@ def tabulate_queryset(self, queryset):
                     "latest_revision": doc.log_entries.last(),
                     "input_by": ";".join(
                         [
-                            str(n)
-                            for n in set(doc.log_entries.values_list("user", flat=True))
+                            user.get_full_name() or user.username
+                            for user in User.objects.filter(pk__in=all_user_ids)
                         ]
                     ),
                     "status": "Public"
                     if doc.status == Document.PUBLIC
                     else "Suppressed",
                     "library": ";".join(
-                        [fragment.collection.lib_abbrev for fragment in all_fragments]
+                        [
+                            fragment.collection.lib_abbrev
+                            for fragment in all_fragments
+                            if fragment.collection and fragment.collection.lib_abbrev
+                        ]
```

Comment:
Ooh, I didn't catch this on the review before — we should use the collection abbreviation if there is no lib abbreviation (and maybe also if there is).  And we should not have any empty values in this field.

I think there should always be at least one abbreviation, but I see it's not required in the model. The default string method for `Collection` uses abbreviations when set, so let's just use that.